### PR TITLE
HAWQ-1261 - add discussion of HAWQ administrative log files

### PR DESCRIPTION
**Pull request by lisakowen**
added a discussion of HAWQ log files.  restructured some existing content.  added new subnav entry.

### CODE REVIEW

File: markdown/admin/logfiles.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+HAWQ administrative log files reside in pre-defined or configured locations on the local file system of the HAWQ node.
+
+**Configuring Logging Parameters**
+
+Configurable logging parameters may affect what, when, and where messages are logged. You configure HAWQ administrative logging options via HAWQ server configuration parameters or command line options.
+
+**Managing Log Files**
+
+Log files are created and/or rotated at pre-defined or configured intervals.   Note that administrative log files are not automatically truncated or deleted. The administrator must implement and periodically run scripts to clean up these log files.
+
```
Comment (dyozie) [inline markdown/admin/logfiles.html.md.erb]: Should this also contain "stub" sections for the "Log format" and "Examining" sections that follow?  maybe just roll the info into the intro paragraph and not into sections here?

File: markdown/admin/logfiles.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+```
+
+### <a id="logfile_examine_db"></a>Examining HAWQ Log Files 
+
+You will examine the HAWQ log files to obtain information about your HAWQ deployment, as well as diagnose problems.
+
+Identify log entries of related transactions using the `transaction_id`. You can identify related log entries of a particular query by the query's session identifier \(`gp_session_id`\) and command identifier \(`gp_command_count`\).
+
+Alternatively, you can use the [*hawq_toolkit*](../reference/toolkit/hawq_toolkit.html#topic16) administrative schema to query HAWQ log files when the [`gp_log_format`](../reference/guc/parameter_definitions.html#gp_log_format) server configuration parameter value specifies `csv` format log files.
+
```
Comment (dyozie) [inline markdown/admin/logfiles.html.md.erb]: This is great info, but are there any example SQL commands we could include or link to from here?

File: markdown/admin/logfiles.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+### <a id="logfile_manage_db"></a>Managing HAWQ Log Files 
+
+HAWQ log output tends to be voluminous, especially at higher debug levels. You should not need to save this information indefinitely. HAWQ administrators typically configure HAWQ to rotate the log files periodically so new log files are created.
+
+Daily log files are created in the `pg_log` subdirectory of the master and each segment data directory. Although log files are rolled over daily, they are not automatically truncated or deleted. Administrators must implement and run scripts to periodically clean up old log files in the `pg_log` directory of the master, standby, and every segment instance.
+
+
+## <a id="mgmtutil_log"></a>Management Utility Log Files 
+You invoke HAWQ management utilities (MU) both from the command line and when you perform cluster management activities using the Ambari user interface. Regardless of source, these utilities log information about command execution status and results. The messages are logged to both `stdout` and a log file.
```
Comment (dyozie) [inline markdown/admin/logfiles.html.md.erb]: Not sure I like the idea of introducing the MU abbreviation.  If this isn't used elsewhere in the docs, let's just spell it out in the titles instead.

Comment (lisakowen) [inline markdown/admin/logfiles.html.md.erb]: removed MU.  as i did this, i ended up changing "utilities" to "utility" in several places...

**Comment by lisakowen** [comment]
made the requested updates.  let me know if you would like to see more hawq_toolkit examples.

**Comment by lisakowen** [comment]
PR merged, closing.

**Pull request closed without merging**